ff/serving/python_client/sdk/transformations.py

```diff
@@ -0,0 +1,49 @@
+transformations_to_register = []
+
+class Transformations:
+    def __init__(self, feature: bool, name: str, variant = str, inputs = list, 
+                        entity = str, fn_type = str, executor = str, frequency = str, ft_type = str):
+        self.feature = feature
+        self.name = name
+        self.variant = variant
+        self.inputs = inputs
+        self.entity = entity
+        self.fn_type = fn_type
+        self.frequency = frequency
+        self.ft_type = ft_type
+
+def transformation(*args, **kwargs):
+
+    def add_transformation(definition_func):
+
+        transformation = Transformations(kwargs['feature'], definition_func.__name__ ,
+                                            kwargs['variant'], kwargs['inputs'], 
+                                            kwargs['entity'], kwargs['fn_type'], kwargs['executor'],  kwargs['frequency'], kwargs['ft_type'])
+        definition_func()
```

Comment:
Why do you run it here?

Reply:
I don't need to run it here, actually. I had been playing around with different implementations of decorators in python and forgot to remove this line. 

We'd run it inside the inner function if we call the transformation decorator like so:
transformation(user_transaction_count)() 

https://www.geeksforgeeks.org/decorators-with-parameters-in-python/